sources/packages/backend/apps/api/src/route-controllers/program-year/program-year.students.controller.ts

```diff
@@ -6,25 +6,33 @@ import {
   ParseIntPipe,
 } from "@nestjs/common";
 import { ProgramYearService } from "../../services";
-import { OptionItem } from "../../types";
+import { ClientTypeBaseRoute } from "../../types";
 import BaseController from "../BaseController";
 import { AllowAuthorizedParty } from "../../auth/decorators/authorized-party.decorator";
 import { AuthorizedParties } from "../../auth/authorized-parties.enum";
-import { ProgramYearDto } from "./models/program-year.dto";
-import { ApiTags } from "@nestjs/swagger";
+import { ProgramYearAPIOutDTO } from "./models/program-year.dto";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
 import { RequiresStudentAccount } from "../../auth/decorators";
+import { OptionItemAPIOutDTO } from "../models/common.dto";
 
 @AllowAuthorizedParty(AuthorizedParties.student)
 @RequiresStudentAccount()
 @Controller("program-year")
-@ApiTags("program-year")
+@ApiTags(`${ClientTypeBaseRoute.Student}-program-year`)
 export class ProgramYearController extends BaseController {
```

Comment:
we also need to add client type in the controller name. I believe that's what we are following 
![image](https://user-images.githubusercontent.com/77353155/208499491-0ad63021-5bc6-4beb-b305-4bb968790df9.png)

Reply:
`ProgramYearStudentsController`